pkg/ui/workspaces/cluster-ui/src/databasesV2/utils.ts

```diff
@@ -36,9 +36,9 @@ export const rawDatabaseMetadataToDatabaseRows = (
     return {
       name: db.db_name,
       id: db.db_id,
-      tableCount: db.table_count,
-      approximateDiskSizeBytes: db.size_bytes,
-      rangeCount: db.table_count,
+      tableCount: db.table_count ?? 0,
```

Comment:
is there any way to avoid having to do stuff like this? When are these undefined?

Reply:
It's for cases where there are no tables in the DB. We could also make sure the API always returns numbers.

Reply:
We can certainly add that on the API - even so I don't think removign this check hurts or should block this PR.
cc @kyle-a-wong Do you want to pick up that task?

Reply:
Oh oops, Yeah these shouldn't have `omitempty` in the struct definition. I think i had goland auto generate these tags and i forgot to remove it. I agree though ,these shouldn't be undefined and should always exist.

fix PR: https://github.com/cockroachdb/cockroach/pull/131441